content/pt/_index.html

```diff
@@ -49,7 +49,7 @@
 
 {{% blocks/feature icon="fas fa-bullseye" title="SDKs" %}}
 <p>Quer desenvolver com facilidade? Utilize nossas SDKs</p>
-<a href="https://docs.openbank.stone.com.br/docs/guias/conta-stone-sdk/overview/"
+<a href="docs/guias/conta-stone-sdk/overview/"
```

Comment:
this content seems to be unavailable - not due to the current change, but I believe we should fix this before merging anyway

Reply:
oh, the right link for this is https://docs.openbank.stone.com.br/sandbox/docs/referencia-da-api/conta-stone-sdk/